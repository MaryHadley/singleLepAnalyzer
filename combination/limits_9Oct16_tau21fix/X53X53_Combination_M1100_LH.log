000 (1295371.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295371.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.237:40876>
...
006 (1295371.000.000) 10/09 22:41:34 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2264  -  ResidentSetSize of job (KB)
...
006 (1295371.000.000) 10/09 22:46:34 Image size of job updated: 283800
	144  -  MemoryUsage of job (MB)
	146896  -  ResidentSetSize of job (KB)
...
006 (1295371.000.000) 10/10 00:01:43 Image size of job updated: 283800
	144  -  MemoryUsage of job (MB)
	146908  -  ResidentSetSize of job (KB)
...
006 (1295371.000.000) 10/10 17:28:30 Image size of job updated: 298064
	157  -  MemoryUsage of job (MB)
	159856  -  ResidentSetSize of job (KB)
...
005 (1295371.000.000) 10/10 17:39:48 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 18:55:42, Sys 0 00:01:04  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 18:55:42, Sys 0 00:01:04  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1594  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1594  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31768        1   3824812
	   Memory (MB)          :      157     3072      3072
...
