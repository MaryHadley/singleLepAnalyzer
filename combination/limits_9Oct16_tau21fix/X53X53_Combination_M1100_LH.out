/var/lib/condor/execute/dir_2135
Total number of likelihood evaluations: 1100000100
Total number of likelihood evaluations: 11000100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_LH/quant-toys:0-sig-71251f107d.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_LH/quant-data-sig-51b370f686.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1100_LH
