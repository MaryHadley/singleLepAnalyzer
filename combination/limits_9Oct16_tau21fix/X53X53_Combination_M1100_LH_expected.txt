# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.083775   0.042266    0.17378   0.05805    0.12265
